saw-support/src/SAWSupport/Pretty.hs

```diff
@@ -33,43 +33,74 @@ Portability : non-portable (language extensions)
 -- prettyprinter, none of which shared any logic other than the
 -- upstream prettyprinter package.
 --
--- Someone should try to systematize the naming. In particular we'd
--- like to have some kind of convention like (perhaps) "prettyFoo"
--- generates a Doc for a Foo and "ppFoo" generates a Doc and also
--- renders it to Text. (Note that in the prettyprinter package,
--- "pretty" generates Doc and there is no "pp", so this choice seems
--- preferable to the other way around.)
+-- XXX: there's more in MrSolver but it will require untangling.
 --
--- I'd prefer to kill off all "show"-related naming because we don't
--- want people writing or using Show instances except maybe for
--- extremely simple types.
+-- XXX: Someone should try to systematize the naming. In particular
+-- we'd like to have some kind of convention like (perhaps)
+-- "prettyFoo" generates a Doc for a Foo and "ppFoo" generates a Doc
+-- and also renders it to Text. (Note that in the prettyprinter
+-- package, "pretty" generates Doc and there is no "pp", so this
+-- choice seems preferable to the other way around.) I've done some
+-- of this, but there's a lot more out in the rest of the tree that
+-- ought to be updated to match it.
+--
+-- XXX: should come up with a better name for pShowText, and remove
+-- pShow entirely. These should be called "show" because they aren't
```

Comment:
Should be called "show"? Or _shouldn't_ be called "show"? I think this was meant to be the latter, but I'm not sure.